src/pages/shorturls.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <Layout>
+    <head>
```

Comment:
The use of Bootstrap here makes the styling weird for the whole page.

In fact, I think you can eliminate the <head> and <body> sections entirely since we don't use these in most of our pages, and Vue/Bulma should take care of everything.